src/pc/djui/djui_hud_utils.h

```diff
@@ -24,7 +24,10 @@ enum DjuiFontType {
 };
 
 struct HudUtilsRotation {
-    f32 rotation;
+    s32 rotationDiff;
+    f32 prevPivotX;
+    f32 prevPivotY;
+    s32 rotation;
```

Comment:
this order seems inconsistent. `s32 rotation;` maybe put it at the beggining of the struct just above `s32 rotationDiff;`

Reply:
I agree